docs/administration-guide/gaffer-config/config.md

```diff
@@ -0,0 +1,150 @@
+## How to configure a Gaffer instance
+
+First you must set up the files and directories you will need for the instance. As it stands there
+are a couple of different ways to run a Gaffer project this example will use a logical structure
+that suites a stand alone deployment consisting of the following file structure:
```

Comment:
Looks like this is using Docker Compose so may want to mention that here.

Reply:
done